transaction_test.go

```diff
@@ -716,7 +716,7 @@ func TestSubmitTransaction_EnvelopeSignature(t *testing.T) {
 
 		script := []byte(`
 		  transaction {
-		    prepare(signer: AuthAccount) {}
+		    prepare() {}
```

Comment:
I think these two should be two tests: (1) signing with an invalid account. (2) signing when the transaction does not expect a signer-parameter.
Maybe keep this test as is, and add a new test for (2)?

Reply:
sure.  i can do that